blindbid.proto

```diff
@@ -20,10 +20,10 @@ message GenerateScoreResponse {
 message VerifyScoreRequest {
     bytes proof = 1;
     bytes score = 2; // BlsScalar
-    bytes seed = 3; // BlsScalar
+    byte seed = 3; // u8
```

Comment:
The comment is redundant since you're sending a `byte` and not `bytes` (the comments where introduced to let a developer knows which kind of structure expects when `bytes` where sent).